kroxylicious-runtime/src/main/java/io/kroxylicious/proxy/config/admin/ManagementConfiguration.java

```diff
@@ -18,12 +17,8 @@
  * @param bindAddress bind address for the management interface. If omitted, all network interfaces will be bound.
  * @param port management port
  * @param endpoints management endpoint configuration
- *
- * <br>
- * Note that {@code host} is accepted as an alias for {@code bindAddress}.  Use of {@code host} is deprecated since 0.11.0
- * and will be removed in a future release.
  */
-public record ManagementConfiguration(@Nullable @JsonAlias("host") @JsonProperty(value = "bindAddress", required = false) String bindAddress,
+public record ManagementConfiguration(@Nullable @JsonProperty(value = "bindAddress", required = false) String bindAddress,
```

Comment:
Wondering if we want a separate note about `host` being removed since it looks like technically it would have worked with the new `management:` style object.